packages/terra-clinical-onset-picker/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Removed react legacy `contexttypes`.
+* Updated the component with `formatMessage` to interface with `react-intl`.
```

Comment:
I think you meant to say `injectIntl` here.

Reply:
I will have it updated. Thank you.